book/src/SUMMARY.md

```diff
@@ -0,0 +1,13 @@
+# Summary
+
+[Introduction](README.md)
+
+# User guide
+
+- [Installation](guide/installation/README.md)
+- [Wallet setup](guide/setup.md)
+
+# Reference guide
+
+- [Command-line tool](cli/README.md)
+  - [start](cli/start.md)
```

Comment:
```suggestion
  - [start](cli/start.md)
  - [example-config](cli/example-config.md)
  - [migrate-zcash-conf](cli/example-config.md)
  - [init-wallet-encryption](cli/init-wallet-encryption.md)
  - [generate-mnemonic](cli/generate-mnemonic.md)
  - [import-mnemonic](cli/import-mnemonic.md)
  - [rpc-cli](cli/rpc-cli.md)
```

I am getting not found in all the command pages except for the start, i think because they are not in the SUMMARY.